### PR TITLE
LPi4a: Update RevyOS to 20240720

### PR DESCRIPTION
**Pull request by wychlw**

### CODE REVIEW

File: LicheePi4A/RevyOS/README_ruyi.md
Comment (KevinMX) [inline LicheePi4A/RevyOS/README_ruyi.md:1]: So do we really need to split one for `ruyi`?
I mean, except flashing procedures.

Comment (wychlw) [reply LicheePi4A/RevyOS/README_ruyi.md:1]: The image in ruyi is outdated for now... So shall we delete it?

Comment (KevinMX) [reply LicheePi4A/RevyOS/README_ruyi.md:1]: > The image in ruyi is outdated for now... So shall we delete it?

Probably yes until it get synced.

**Review by KevinMX — APPROVED** [review]
LGTM.

**Pull request merged**